developer/guides/troubleshooting/layout-shift.mdx

```diff
@@ -77,6 +77,23 @@ React used to log a warning for this situation when using the Next.js Pages Rout
 - **Match SSR & Client:** Ensure initial props/state are identical. For example, compute or fetch values during SSR so the client sees the same markup.
 - **Memoize Values:** Wrap dynamic context or prop values in [`useMemo`](https://react.dev/reference/react/useMemo) to avoid identity changes during hydration.
 - **Defer Non-Critical Updates:** Use React's [`startTransition`](https://react.dev/reference/react/startTransition) to mark updates as non-urgent, preventing fallback rendering.
+- **Use Deferred Values:** Apply [`useDeferredValue`](https://react.dev/reference/react/useDeferredValue) to postpone non-urgent context or prop updates until hydration completes. Using `useDeferredValue` allows the client to finish hydrating before React propagates the new value, avoiding any intermediate fallback UI and eliminating the layout shift. Example:
```

Comment:
The last two bullet points are really the same. How about:

- **Defer Non-Critical Updates:** Use React's [`startTransition`](https://react.dev/reference/react/startTransition) and [`useDeferredValue`](https://react.dev/reference/react/useDeferredValue) to mark updates as non-urgent and prevent fallback rendering. For example, if a layout shift occurs because a context provider’s value changes between server-side rendering and client-side rendering, wrap that value with `useDeferredValue`. This allows the client to finish hydrating before React propagates the updated value, avoiding intermediate fallback UI and eliminating the layout shift:

Reply:
Sure, updated.